src/android/toga_android/widgets/base.py

```diff
@@ -64,4 +84,17 @@ def add_child(self, child):
             child.container = self.container
 
     def rehint(self):
+        # An Android `View` subclass (of which all widgets are subclasses) can
+        # be measured against a measurement specification called a MeasureSpec.
+        # Note that this mutates the widget's `getMeasuredWidth()` and
+        # `getMeasuredHeight()`.
+        self.native.measure(
+            View__MeasureSpec.UNSPECIFIED, View__MeasureSpec.UNSPECIFIED
+        )
+        # We use `at_least()` below because without it, Travertino seems to
+        # decide to use a width of zero.
+        self.interface.intrinsic.width = at_least(self.native.getMeasuredWidth())
+        self.interface.intrinsic.height = self.native.getMeasuredHeight()
+
+    def set_size(self, size):
```

Comment:
Is this a leftover from something else? It doesn't appear to be otherwise needed.

Reply:
Totally leftover. I'll remove immediately in a follow-up.

Reply:
Done.